tests/data/comments2.py

```diff
@@ -142,7 +142,7 @@ def inline_comments_in_brackets_ruin_everything():
         syms.simple_stmt,
         [
             Node(statement, result),
-            Leaf(token.NEWLINE, '\n'),  # FIXME: \r\n?
+            Leaf(token.NEWLINE, '\n')  # FIXME: \r\n?
```

Comment:
What caused this change? Black should put a trailing comma here.

Reply:
You're right, but you're commenting on the pre-formatted part of the test case, not the result. The spirit of the test appeared to be that the comment moved correctly with the collapsed line, so I removed the trailing comma in the input.

Reply:
Thanks! Sorry for that, it's not always easy to tell from changes in the test suite what is actually changing.